src/SyntheticGrids.jl

```diff
@@ -73,7 +79,7 @@ load_grid
 import Base.==
 import Base.show
 import Base.hash
-import Base.mean
+import Statistics.mean
```

Comment:
I would just do this in `src/SyntheticGrids.jl` and remove the `using` statements from the other files.
```suggestion
using Statistics: Statistics, mean
```